test/proposals/proposals.jl

```diff
@@ -32,3 +33,14 @@ end
 
 @testset "RWalk" begin
 end
+
+@testset "RStagger" begin
+    # printing
+    prop = Proposals.RStagger()
+    prop2 = Proposals.RStagger(ratio=0.1, walks=20)
+    @test prop.ratio == 0.5
+    @test prop.walks == 25
+    @test prop.scale == 1
+    @test prop2.ratio == 0.1
+    @test prop2.walks == 20
```

Comment:
Add some assertion tests here to match up with the new struct definition-

```julia
@test_throws AssertionError Proposals.RStagger(ratio=-0.2)
...
```

See the tests for `RWalk` to get a good idea (you can probably just copy-paste them).

Reply:
done